### ISSUE TITLE
[Gssoc]: feat: Contact us Backend 

### ISSUE THREAD

**Issue by ManpreetSingh18** [oxiton-foundation/click-metrics#187]
### Description

I will create a model, controller, and route for the Contact Us page to store data in MongoDB Cloud (Atlas) and connect with the front end.

### Changes You need to make shown In ss
![Screenshot 2024-06-20 140148](https://github.com/oxiton-foundation/click-metrics/assets/72294811/fb3e7ba5-0b5f-4e00-9829-fce708ec8888)

### Screenshots

![image](https://github.com/oxiton-foundation/click-metrics/assets/72294811/2c649765-5ffd-42c9-8ca8-b8fdcb04c804)

### Checklist

Before submitting your pull request, ensure that you have completed the following tasks:

- [  ✔] Gone through the [rules and guidelines](https://github.com/oxiton-foundation/click-metrics/issues/24) of the project.
- [ ✔ ] When you are creating PR, your PR should title follows the specified format: [Feature/Enhancement/Bug Fix]: Brief description of the change.
- [  ✔] Attached OSSI or GSSoC in the heading of the Issue.
- [  ✔] I have checked this issue is not already covered in another issue.

### Expected Time to Complete

I will complete it in 1.5 days or earlier maybe.

**Comment by Swarnendu0123** [comment]
What do you want to achieve? Please clearly explain in terms of functionality.

**Comment by ManpreetSingh18** [comment]
I want to create a Backend for it to store the details of users who fill Help form or contact us form in mongo Db database.

**Comment by Swarnendu0123** [comment]
How the feature will work? Is the contact accessible by admin palate only? Are you going to design the admin palate also?

**Comment by ManpreetSingh18** [comment]
Yes, only the admin can access that data or the one having access to the database.I can also make an admin palate if you say . For that, I will develop particular separate /admin endpoints and have a login system to check for authentication. Below is the sample example I have made for someone else
![Screenshot 2024-06-21 154805](https://github.com/oxiton-foundation/click-metrics/assets/72294811/eeac9d42-f8db-4b5a-8f79-501aedfd0554)

**Comment by Swarnendu0123** [comment]
render the contacts in a table that will be better!

**Comment by ManpreetSingh18** [comment]
@Swarnendu0123 In this issue am I supposed to use Firebase na?

**Comment by Swarnendu0123** [comment]
yes!

**Comment by ManpreetSingh18** [comment]
Fixed  by #204
